src/error.rs

```diff
@@ -44,25 +43,8 @@ impl From<udev::Error> for Error {
 
 impl fmt::Display for Error {
 	fn fmt(&self, f: &mut fmt::Formatter) -> Result<(), fmt::Error> {
-		f.write_str(error::Error::description(self))
+		f.write_str(format!("{}", self).as_str())
 	}
 }
 
-impl error::Error for Error {
```

Comment:
This seems to be removing the `std::error::Error` impl completely, which is not only a breaking change, but also bad as all errors should implement that trait.

Reply:
Yes, forget the salt, you are absolutely right. I did not think that one trough. Let me fix that.

Reply:
Would you be so kind to take a look whether the current changes would be acceptable (even if you are not the maintainer).